src/main/resources/stock_trades_schema.avro

```diff
@@ -0,0 +1,64 @@
+{
+  "namespace": "ksql",
+  "name": "StockTrade",
+  "doc": "Defines a hypothetical stock trade using some known simulated stock symbols.",
+  "type": "record",
+  "fields": [
+    {
+      "name": "side",
+      "doc": "A simulated trade side (buy or sell or short)",
+      "type": {
+        "type": "string",
+        "arg.properties": {
+          "options": [ "BUY", "SELL", "SHORT" ]
+        }
+      }
+    },
+    {
+      "name": "quantity",
+      "doc": "A simualted random quantity of the trade",
```

Comment:
1. "simualted" --> "simulated"
2. All of this is simulated, so it is redundant to say it everywhere

Reply:
1. Thank you, will fix typo
2. With finance, it's my inclination to be overly abudant with explicitness about the fact that we are simulating things here...